nova/compute/api.py

```diff
@@ -362,7 +362,8 @@ def _check_num_instances_quota(self, context, instance_type, min_count,
         try:
             quotas = objects.Quotas(context)
             quotas.reserve(context, instances=max_count,
-                           cores=req_cores, ram=req_ram)
```

Comment:
这个地方上游没有带context，是因为上游在修复这个bug之前已经修复了多个提交，其中一个提交中就是将quotas.reserve(context,
instances=max_count 改为quotas.reserve( instances=max_coun
去掉了context，见（4031272fe93d1046cff4c0cc788c8e78db944419）
在修复该bug的目的是admin
做restore没有带project_id与user_id，因此恢复默认使用了admin来进行恢复，因此在不做大的改动原则下，添加了project_id,
user_id ，从目前测试来看没有发现问题．

2016-07-08 11:08 GMT+08:00 daemon <notifications@github.com>:

> In nova/compute/api.py
> <https://github.com/eayunstack/nova/pull/37#discussion_r70020387>:
>
> > @@ -362,7 +362,8 @@ def _check_num_instances_quota(self, context, instance_type, min_count,
> >          try:
> >              quotas = objects.Quotas(context)
> >              quotas.reserve(context, instances=max_count,
> > -                           cores=req_cores, ram=req_ram)
>
> 这里上游没有带context参数
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/eayunstack/nova/pull/37/files/b9f59ddec6dfb4785ef13559f31c37d1239b21e7#r70020387>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AKMiZlsZ4KvSLVGnDuGxCWpzLS1daYTRks5qTb8UgaJpZM4I0Frh>
> .
>